bwi_services/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(bwi_services)
+
+find_package(catkin REQUIRED COMPONENTS 
+  roscpp
+  rospy
+  std_msgs
+  message_generation 
+  bwi_kr_execution
+  actionlib
+)
+
+add_service_files(
+  FILES
+  DeliverMessage.srv
+  GoBesideLocation.srv
+  GoToLocation.srv
+  SpeakMessage.srv
+)
```

Comment:
SpeakMessage is text-to-speech. DeliverMessage goes to a location and then says the message.